stellar/docs/link-token.md

```diff
@@ -0,0 +1,285 @@
+# Link Token Documentation
+
+## Overview
+
+This document explains how to link custom tokens on Stellar to other chains using the Interchain Token Service (ITS).
+
+For detailed design specifications and architecture, see **ARC-1: ITS Hub Multi-Chain Token Linking**:
+https://github.com/axelarnetwork/arcs/blob/031ec16a700efe166a727d5ae8909a39f7c6ae13/ARCs/ARC-1.md # skip-check
+
+The token linking feature enables:
+
+- Linking custom tokens deployed on Stellar with tokens on EVM chains
+- Supporting tokens with different decimal precisions through automatic scaling
+
+## How It Works
+
+The token linking process involves two key message types that ITS Hub uses to coordinate the linking:
+
+### Token Metadata Registration
+
+Before linking tokens, ITS Hub needs to know about each token's details (address and decimals) on both chains.
+
+### Token Linking
+
+Once metadata is registered, you can link tokens by specifying which tokens on different chains should be connected.
+
+## Link Token Flow
+
+```
+1. User controls Token A (Stellar) and Token B (EVM)
+2. User → ITS Stellar: registerTokenMetadata(Token A)
+3. User → ITS EVM: registerTokenMetadata(Token B)
+4. User → ITS Stellar: registerCustomToken() → Deploys Token Manager A
+5. User → Verify token metadata is registered on ITS Hub
+6. User → ITS Stellar: linkToken() → Deploys Token Manager B on destination chain
+7. User → Transfer or add mintership to the Token Manager (MINT_BURN type only)
+8. Token linking complete - InterchainTransfer enabled
+```
+
+## Prerequisites
+
+Before linking tokens, ensure you have:
+
+1. **Token Control**: Token permissions depend on the manager type:
+    - **MINT_BURN**: You must have minter permissions for the token on that chain
+    - **LOCK_UNLOCK**: No token control or minter permissions required for that token
```

Comment:
I dont think permission is required, but I believe their needs to be an `allowance` set for this manager type as it calls `transferFrom()` on the token, which I think is worth calling out.